src/EventListener/ReadListener.php

```diff
@@ -56,10 +59,12 @@ public function onKernelRequest(GetResponseEvent $event)
             return;
         }
 
-        if (isset($attributes['collection_operation_name'])) {
-            $data = $this->getCollectionData($request, $attributes);
-        } else {
-            $data = $this->getItemData($request, $attributes);
+        foreach (OperationTypes::TYPES as $operationType) {
+            if (isset($attributes["{$operationType}_operation_name"])) {
```

Comment:
Actually, it can be very dangerous without brackets: http://php.net/manual/en/language.types.string.php#language.types.string.parsing